wire/src/main/java/org/ethereum/beacon/wire/SimplePeerManagerImpl.java

```diff
@@ -86,6 +93,15 @@ protected HelloMessage createLocalHello() {
         head.getBlock().getSlot());
   }
 
+  private void updateBestSlot(PeerImpl peer) {
```

Comment:
I don't understand, who else should care about best slot of all peers? Manager. I've chose `SimplePeerManagerImpl` because we are not required to start sync to know it. Separate `MaxKnowBestSlotManager`, really? It looks like over-engineering.

Reply:
This is single responsibility. I think there should be a component that accumulates knowledge about the chain we're currently connected to. It could be not only max known slot. You may leave it as it is for now. It would be just cleaner if peers management and chain attributes were separated.